ui/apps/ui/src/app/pages/search-page/search-page.component.html

```diff
@@ -96,7 +96,7 @@
 
 <!--          </div>-->
 
-          <a href="{{marketplaceUrl}}user_documentation" class="right-media-link">
+          <a href="https://docs.sandbox.eosc-beyond.eu/" class="right-media-link">
```

Comment:
It would be better to create a separate variable for the documentation URL instead of hard-coding the link. This makes the code more flexible and easier to maintain across different environments